pytorch_lightning/plugins/training_type/ddp2.py

```diff
@@ -28,7 +28,7 @@ def global_rank(self) -> int:
     def world_size(self) -> int:
         return self.num_nodes
 
-    def setup(self, model):
+    def setup_model(self, model):
```

Comment:
I'd rather not rename these, since they aren't meant to setup only the model. They can actually setup arbitrary things